styles.css

```diff
@@ -22,27 +23,158 @@ nav {
 }
 a {
 	text-decoration: none;
-	color: #D685FF;
+	color: #9fa0ff;
 	cursor: pointer;
+	width: 50px;
+}
+a:hover {
+  color: #FB88BC;
+}
+header {
+	margin: 25% 10%;
+	padding: 50px;
+	text-align: center;
+	color: #9fa0ff;
+	font-size: 16pt;
+	background-image: url('clouds.png');
+	background-size: contain;
+	background-repeat: no-repeat;
+	background-position: center;
+}
+h1 {
+	margin-top: 20px;
+}
+.email {
+	color: #3E3B3F;
+}
+.about {
+	margin: 10%;
+	margin-top: 0;
+	margin-bottom: 20px;
+	color: #3E3B3F;
+	line-height: 1.6;
+	font-size: 13pt;
+}
 
+h2 {
+	margin: 10%;
+	margin-top: 100px;
+	margin-bottom: 10px;
+	color: #FB88BC;
+	font-size: 30pt;
+}
+article {
+	margin: 10% 10%;
+	color: #3E3B3F;
+	line-height: 1.6;
+	font-size: 13pt;
+	display: flex;
+	align-items: flex-start;
+}
+img {
+	border-radius: 8px;
+	max-width: 45%;
+	height: auto;
+}
+.projectInfo {
+	margin-right: 40px;
+	color: #FB88BC;
+}
+.projectTitle {
+	font-weight: bold;
+	width: 90px;
+	cursor: pointer;
+}
+.projectDescription {
+	max-width: 250px;
+	min-width: 150px;
+	margin: 0 20px;
+}
+.textContent p {
+	margin-top: 0;
+	margin-bottom: 0;
+}
+.textContent {
+	display: flex;
+	justify-content: space-between;
+}
+.foot {
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    color: white;
+    text-align: center;
+	display: flex;
```

Comment:
indentation here and the line below

Reply:
that's strange it isn't indented on mine